composables/useItemInteraction.ts

```
@@ -62,13 +69,57 @@ export function useItemInteraction(
+    }
+    // Store initial positions when group movement starts
+    if (kind === "group" && itemId) {
+      // Reset the initial positions object
+      selectedItemsInitialPositions.value = {};
+
+      // Find the group item
+      const groupItem = boardStore.board?.data.items.find(
+        (item) => item.id === itemId && item.kind === "group"
+      );
+
+      if (groupItem && groupItem.content && groupItem.content.itemIds) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)